foxpuppet/tests/test_browser_model.py

```diff
@@ -0,0 +1,57 @@
+from foxpuppet import FoxPuppet
+
+
+class Test_Browser_Window:
+
+    # Tests minimizing the browser window
+    def test_minimize_browser_window(self, windows, selenium):
+
+        FoxPuppet(selenium)
+        selenium.get('http://www.google.com')
+        selenium.set_context('chrome')
+        windows.min_window_size()
+        windows.close_all()
+        selenium.quit()
```

Comment:
There's no need to call `selenium.quit()` as pytest-selenium will do that for you. I think we can simplify this by using a `foxpuppet` fixture. Personally I'd prefer the test to look something like:

```python
def test_minimize_window(foxpuppet):
    foxpuppet.window.minimize()
    # assert something
```

There's no need to open a website (this also makes our tests dependent on that website being available, and having a network connection). I also think that `FoxPuppet` should provide access to the window directly rather than having a separate `Browser_Window` object to instantiate. Finally, the switching of context should be masked from the user and handled in the FoxPuppet methods.